src/app/ApplicationTemplate.UITests/Tests/StartupTests.cs

```diff
@@ -9,10 +9,10 @@ namespace ApplicationTemplate.UITests.Tests
 	[TestFixture]
 	public class StartupTests : TestBase
 	{
-		[Test]
-		public void ShowRepl()
-		{
-			TestBase.App.Repl();
-		}
+		//[Test]
```

Comment:
same

Reply:
Added a TODO task #. Essentially the UI tests don't seem to run well on Azure Pipelines.